CalPatRec/inc/CalHelixFinderData.hh

```diff
@@ -50,9 +50,9 @@ namespace mu2e {
     //   // kNPanelsPerFace =  3
     // };
 
-    enum { kMaxResidIndex = 500 };
+    enum { kMaxResidIndex = 5000 };
 
-    constexpr static uint16_t        kNMaxChHits = 500;
+    constexpr static uint16_t        kNMaxChHits = 5000;
```

Comment:
This change is acceptable as a short-term bug fix, but it opens questions about implications to performance.

As a possible long-term fix, can the storage associated with this value:
Offline/CalPatRec/inc/CalHelixFinderData.hh:      float    resid[kMaxResidIndex];
Offline/CalPatRec/inc/CalHelixFinderData.hh:      float    dist [kMaxResidIndex];
Offline/CalPatRec/inc/CalHelixFinderData.hh:      float    dz   [kMaxResidIndex];
Be converted to std::vector instead?

Reply:
I remember a long time ago that using an array instead of vector was visibly making a difference in the timing performance when I was testing the trigger reconstruction.